pkg/metrics/server.go

```diff
@@ -42,9 +43,15 @@ func NewServer(subsystem Subsystem) *Server {
 
 	var secureMetricsServer *http.Server
 	if secureMetricsEnabled() {
+		tlsConfig, err := tlsConfigurer.TLSConfig()
+		if err != nil {
+			utils.Should(errors.Wrap(err, "failed to create TLS config loader"))
```

Comment:
Let's keep it consistent and also add a warning log that the secured metrics won't be served.